### DISCUSSION TITLE
setting up HASPone firmware update brick

### DISCUSSION THREAD

**Discussion by radionerd1** [HASwitchPlate/HASPone#57] [Q&A]
I started setting up my HASPone, hit the firmware update in the web gui. It successfully updated, then I entered my MQTT info, Saved, and then it went into a boot loop, lost display. 
I was able to recover by flashing the D1 mini chip using Tazmotizer. "HASwitchPlate.ino.d1_mini" brought it back to default settings.

log after loop:
`--------------- CUT HERE FOR EXCEPTION DECODER ---------------

 ets Jan  8 2013,rst cause:1, boot mode:(3,7)

load 0x4010f000, len 3584, room 16 
tail 0
chksum 0xb0
csum 0xb0
v2843a5ac
~ld

================================================================================
[+0.073s] SYSTEM: Starting HASwitchPlate v1.00
[+0.079s] SYSTEM: Last reset reason: Fatal exception:0 flag:2 (Exception) epc1:0x4000e25d epc2:0x00000000 epc3:0x00000000 excvaddr:0x00000000 depc:0x00000000
[+0.093s] SYSTEM: heapFree: 31496 heapMaxFreeBlockSize: 31392
Crash information recovered from EEPROM
Crash # 1 at 450 ms
Restart reason: 2
Exception cause: 0
epc1=0x4000e25d epc2=0x00000000 epc3=0x00000000 excvaddr}0x00000000 depc=0x00000000
>>>stack>>>
3ffffea0: 3fff50dc 3fffff20 40238910 00000000 
3ffffeb0: 4022f3ae 40101227 3fff1a9c 4022f4b5 
3ffffec0: 0000001c 00000000 00000000 00000000 
3ffffed0: 00000003 4022f84d ffffffff 00000001 
3ffffee0: 3ffe8510 00000001 3fff0b80 3f800000 
3ffffef0: 00000000 00000001 3fff0b80 4022a1f8 
3fffff00: 00000000 3fff1a00 000d000f 3f800000 
3fffff10: 00000000 0000001c 00000000 4023c974 
3fffff20: 3fffdad0 3ffe8510 3fff0b80 402027b8 
3fffff30: 3fffdad0 3fffff60 3fff0b60 4021509e 
3fffff40: 3fff1a00 0052005f 80efeffe feefeffe 
3fffff50: 3fff1a00 0017001f 80efeffe feefeffe 
3fffff60: 3fff4500 0033003f 80efeffe feefeffe 
3fffff70: 39343100 feef0036 80efeffe 3fff1a00 
3fffff80: 0012001f 80efeffe feefeffe feefeffe 
3fffff90: feefeffe feefeffe feefeffe 3fff0cdc 
3fffffa0: 3fffdad0 00000000 3fff0c9c 4022d068 
3fffffb0: feefeffe feefeffe 3ffe8588 40100701 
<<<stack<<<
EEPROM space available: 0x00bb bytes
================================================================================

[+0.287s] SPIFFS: mounting SPIFFS
[+0.306s] SPIFFS: reading /config.json
[+0.316s] SPIFFS: read 354 bytes and parsed json:{"mqttServer":"192.168.1.220","mqttPort":"1883","mqttUser":"mqtt","mqttPassword":"mypasswrd","mqttTlsEnabled":false,"mqttFingerprint":"","haspNode":"plate01","groupName":"plates","configUser":"admin","configPassword":"","nextionBaud":"","motionPinConfig":"0","debugSerialEnabled":true,"debugTelnetEnabled":false,"mdnsEnabled":true,"beepEnabled":false}
`

**Comment by ielbury** [comment]
I have seen this as well after the first flash. I surmised that it was due to the SPIFFS file system not being formatted after it was first flashed. After a reboot everything was fine for me.

Does it still log the same error after a reboot?

**Reply by radionerd1** [reply]
I flashed it at back to life at "Oh Dark 30" last night. I think it is fine now. It woke up on the initial screen with QR code. Will play with it soon.

I wonder if it borked since I didn't reboot after clicking update firmware. It went dark after my next step when I saved the MQTT settings, after update.

Thanks
